include/libtorrent/settings_pack.hpp

```diff
@@ -1045,8 +1048,15 @@ namespace libtorrent
 			// torrents.
 			//
 			// ``active_limit`` is a hard limit on the number of active (auto
-			// managed) torrents. This limit also applies to slow torrents.
-			//
+			// managed) torrents. This limit also applies to slow torrents and it
+			// does apply to all announcement protocols as well. If you have a
```

Comment:
this term is not obvious: "announcement protocols as well". I take it you refer to the limits on where to announce torrents. Maybe there's a more concise way to say that. Perhaps you could just drop " protocols as well" (since the start of the sentence is "This limit also applies to"). I would also shorten: "and it does apply to all announcements" -> "and to announcements"